examples/spark/README.md

```diff
@@ -94,70 +98,53 @@ $ kubectl create -f examples/spark/spark-worker-controller.json
-                                                                    4 (0  6.8 GB
-worker-20150318151746-192.168.90.17-50490 192.168.90.17:50490 ALIVE Used) (0.0 B
-                                                                          Used)
-...
+$ kubectl get pods
+NAME                                           READY     REASON    RESTARTS   AGE
+[...]
+spark-master                                    1/1       Running   0          14m
+spark-worker-controller-hifwi                   1/1       Running   0          33s
+spark-worker-controller-u40r2                   1/1       Running   0          33s
+spark-worker-controller-vpgyg                   1/1       Running   0          33s
+
+$ kubectl logs spark-master
+[...]
+15/06/26 14:15:43 INFO Master: Registering worker 10.244.2.35:46199 with 1 cores, 2.6 GB RAM
+15/06/26 14:15:55 INFO Master: Registering worker 10.244.1.15:44839 with 1 cores, 2.6 GB RAM
+15/06/26 14:15:55 INFO Master: Registering worker 10.244.0.19:60970 with 1 cores, 2.6 GB RAM
 ```
-
-(Pull requests welcome for an alternative that uses the service IP and
-port)
-
 ## Step Three: Do something with the cluster
 
-```shell
-$ kubectl get pods,services
-POD                             IP                  CONTAINER(S)        IMAGE(S)             HOST                          LABELS                                STATUS
-spark-master                    192.168.90.14       spark-master        mattf/spark-master   172.18.145.8/172.18.145.8     name=spark-master                     Running
-spark-worker-controller-51wgg   192.168.75.14       spark-worker        mattf/spark-worker   172.18.145.9/172.18.145.9     name=spark-worker,uses=spark-master   Running
-spark-worker-controller-5v48c   192.168.90.17       spark-worker        mattf/spark-worker   172.18.145.8/172.18.145.8     name=spark-worker,uses=spark-master   Running
-spark-worker-controller-ehq23   192.168.35.17       spark-worker        mattf/spark-worker   172.18.145.12/172.18.145.12   name=spark-worker,uses=spark-master   Running
-NAME                LABELS                                    SELECTOR            IP                  PORT
-kubernetes          component=apiserver,provider=kubernetes   <none>              10.254.0.2          443
-spark-master        name=spark-master                         name=spark-master   10.254.125.166      7077
-
-$ sudo docker run -it mattf/spark-base sh
+Get the address and port of the Master service.
 
-sh-4.2# echo "10.254.125.166 spark-master" >> /etc/hosts
+```shell
+$ kubectl get service spark-master
+NAME           LABELS              SELECTOR            IP(S)          PORT(S)
+spark-master   name=spark-master   name=spark-master   10.0.204.187   7077/TCP
+```
 
-sh-4.2# export SPARK_LOCAL_HOSTNAME=$(hostname -i)
+SSH to one of your cluster nodes and run spark-base image. Inside of the image there is a script
+that sets up the environment besed on the provided ip and port of the Master. After running it
+you are good to go. 
```

Comment:
I think it would be more in line with how Kubernetes should be used—`ssh`-ing into a node doesn't seem like a workflow we should encourage our users to do.  I'd recommend putting it in prerequisites; "You need [an installation of Docker](https://docs.docker.com/installation/) to run this example."  Bump to @RichieEscarez for a third set of eyes.